cypress/e2e/po/pages/users-and-auth/users-and-auth.po.ts

```diff
@@ -0,0 +1,78 @@
+import PagePo from '@/cypress/e2e/po/pages/page.po';
+import BaseResourceList from '@/cypress/e2e/po/lists/base-resource-list.po';
+import LabeledInputPo from '@/cypress/e2e/po/components/labeled-input.po';
+import LabeledSelectPo from '@/cypress/e2e/po/components/labeled-select.po';
+import AsyncButtonPo from '@/cypress/e2e/po/components/async-button.po';
+import CheckboxInputPo from '~/cypress/e2e/po/components/checkbox-input.po';
+
+export default class UsersAndAuthPo extends PagePo {
+  static url: string = '/c/_/auth/management.cattle.io.user'
+
+  static goTo(): Cypress.Chainable {
+    return super.goTo(UsersAndAuthPo.url);
+  }
+
+  constructor(url: string) {
+    super(url || UsersAndAuthPo.url);
+  }
+
+  listCreate() {
+    const baseResourceList = new BaseResourceList(this.self());
+
+    return baseResourceList.masthead().actions().eq(0).click();
+  }
+
+  listElements() {
+    const baseResourceList = new BaseResourceList(this.self());
+
+    return baseResourceList.resourceTable().sortableTable().rowElements();
+  }
+
+  listElementWithName(name: string) {
+    const baseResourceList = new BaseResourceList(this.self());
+
+    return baseResourceList.resourceTable().sortableTable().rowElementWithName(name);
+  }
+
+  listTitle() {
```

Comment:
This should go in `ResourceListMastheadPo` (as it's part of the masthead component)

Reply:
Updated. Thanks for the input